hyperspy/tests/io/test_edax.py

```diff
@@ -0,0 +1,217 @@
+import nose.tools as nt
+
+import gzip
+import hashlib
+import os.path
+import os
+import shutil
+import numpy as np
+
+from hyperspy.io import load
+from hyperspy import signals
+
+test_files = ['Live Map 2_Img.ipr',
+              'single_spect.spc',
+              'spd_map.spc',
+              'spd_map.spd']
+my_path = os.path.dirname(__file__)
+
+
+class TestSpcSpectrum:
+
+    def setUp(self):
+        print('testing single spc spectrum...')
+        self.spc = load(os.path.join(
+            my_path,
+            "edax_files",
+            test_files[1]))
+
+    def test_data(self):
+        nt.assert_equal(np.uint32, self.spc.data.dtype)     # test datatype
+        nt.assert_equal((4096,), self.spc.data.shape)       # test data shape
+        nt.assert_equal(
+            [0, 0, 0, 0, 0, 0, 1, 2, 3, 3, 10, 4, 10, 10, 45, 87, 146, 236,
+             312, 342], self.spc.data.tolist()[:20])   # test 1st 20 datapoints
```

Comment:
A minor thing, but I would swap the `tolist()` and slicing operations - no need to create a list with all 4096 elements